tor/core/admin_commands.py

```diff
@@ -89,17 +146,15 @@ def process_blacklist(reply, config):
             failed.append(username)
             continue
 
-        already_added = config.redis.sadd('blacklist', username)
-        if already_added == 0:
+        redis_already_added = config.redis.sadd('blacklist', username)
+        if redis_already_added == 0:
```

Comment:
Looks like we're only using `already_added`/`redis_already_added` once. What about something simpler, like below?

```python
if config.redis.sadd('blacklist', username):
    pass
```

The return for `redis.sadd()` is just a boolean-like response (by way of integers, where `0` is false and `1` is true) saying if it was successful in adding a value to a set. Python evaluates `0` as false and `1` as true as well. Can we not just rely on that?

Reply:
Huh, TIL.